src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1282,9 +1294,9 @@ static void addBsonFilter
   }
   else
   {
-    std::string details = std::string("unknown query operator: ") + opr;
+    std::string details = std::string("unknown query operator: '") + opr + "'";
```

Comment:
Is this `details` used in some way in the response payload to client? Note that `'` is a forbidden char....